### ISSUE TITLE
Stage - Inline menu

### ISSUE THREAD

**Issue by edenvidal** [rnbwdev/rnbw#591]
- [ ] the "/" menu is a quick way to add things. thinks of it as the "add" menu, but inline - when you type "/" in a text field, a dropdown menu appears. this menu shows different types of content you can add, like text, headings, lists, images, or embeds.
- [ ] you can scroll through the options or start typing to filter and find what you need quickly. for example, typing "/image" will give

![Image](https://github.com/rnbwdev/rnbw/assets/13418616/d15cad33-9be0-433a-8c25-3cdbcefb21f9)

**Comment by atulbhatt-system32** [comment]
@edenvidal For now we don't have any direct typing experience in stage. We can only type when in content-editable mode. So how do you think we should carry on with the "/" menu? Shall we just pop up a menu like cmdk menu shows?

**Comment by edenvidal** [comment]
@atulbhatt-system32 can't we display the menu on stage? like for every time someone hit "/" ?

**Comment by atulbhatt-system32** [comment]
we can do it like cmd+k

**Comment by edenvidal** [comment]
@atulbhatt-system32, you're right. It differs from pressing A. the justification is the experience of adding content without context switching. 

It is probably worth mentioning that:
- the exact representation of the same data fuels the "a" menu.
- no "a" is available while writing content, so "/" works in this case.
- everything that comes after the "/" is a search/filter for the inline menu.
